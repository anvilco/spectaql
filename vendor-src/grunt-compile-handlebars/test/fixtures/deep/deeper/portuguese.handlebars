<h2>{{saudacao}}{{pontuacao}} {{localizacao}}</h2>
