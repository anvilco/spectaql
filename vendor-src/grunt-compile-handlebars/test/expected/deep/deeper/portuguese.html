<h2>Ola, mundo</h2>
